tests/ui/const-generics/const_patam_ty_impl_bad_field.rs

```diff
@@ -0,0 +1,12 @@
+#![feature(const_param_ty_trait)]
+
+#[derive(PartialEq)]
+struct NotParam;
+
+#[derive(PartialEq)]
```

Comment:
Should be `#[derive(PartialEq, Eq)]` (same for `NotParam`) since `derive(PartialEq)` isn't really meaningful for this test. It doesn't add a `StructuralEq` impl and the `PartialEq` isn't used anywhere.

Reply:
Fixed